src/ThreeEditor/Simulation/CustomStoppingPower/CustomStoppingPower.ts

```diff
@@ -15,15 +15,18 @@ export interface StoppingPowerFile {
 	content: string;
 }
 
-const getAvailableStoppingPowerFiles = (path: StoppingPowerTable) => {
+const getAvailableStoppingPowerFiles = async (path: StoppingPowerTable) => {
```

Comment:
that code is copy-pasted from https://threejs.org/ (or https://github.com/mrdoob/three.js/)

maybe instead of adjusting the code yourself you go to https://github.com/mrdoob/three.js/ and ask mrdoob what he thinks of using his library in vite ?  

you will achieve more with communication than with coding !